ting_material_details.field.inc

```diff
@@ -457,6 +463,36 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
             '#markup' => $entity->serieDescription,
           );
           break;
+
+        case 'ting_details_is_part_of_search':
+          $type = $uri = NULL;
+          foreach ($entity->getRelations() as $relation) {
+            $type = explode(':', $relation->getType());
+            if ( isset($type[1]) && $type[1] == 'isPartOfManifestation' ) {
+              $uri  = $relation->getURI();
+            }
+          }
+          $render['#prefix'] = '<span class="ting-material-detail ting-material-is-part-of">';
+          $render['#suffix'] = '</span>';
+          if (!is_array($val)) {
+            $val = array($val);
+          }
+          if ( $uri ) {
+            foreach ($val as $key => $detail) {
+              $render['part_of'][$key]['#theme'] = 'link';
+              $render['part_of'][$key]['#text'] = $detail;
+              $render['part_of'][$key]['#path'] = 'search/ting/' . $uri;
```

Comment:
Thanks for elaborating. If the intent is to create *one* link I would expect to code to reflect this.

The conversion to an array value (L477-479) makes it appear as if a non-array value is an exception to be handled and multiple links are the end goal. If that is not the case I would expect the code to reflect this.

I appreciate if this is partly in preparation for #1109 but then I would expect the preparation to extend to URI extraction. The current state matches an array with a single value.